content/guides/building/get-source-git.md

```diff
@@ -40,141 +44,150 @@ git clone https://review.haiku-os.org/buildtools
+<h4>Build Tools:</h4>
 
 ```sh
-git config --global user.name "John Doe"
-git config --global user.email "john.doe@developers.com"
+git clone http://github.com/haiku/buildtools.git
 ```
 
-On macOS, you must set the following option in order to avoid problems with the unicode representation of filenames:
+<h4>Haiku:</h4>
 
 ```sh
-git config core.precomposeunicode true
+git clone http://github.com/haiku/haiku.git
 ```
 
-<h4>Setup an account on Gerrit</h4>
-
-<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. You currently need a <a href="https://github.com">Github</a> account for logging in.</p>
-
-<div class="alert alert-warning">
-Make sure your ssh key is generated with ed25519. RSA keys will not work with Gerrit.
-</div>
-<p>Upload your SSH public key in <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
-<p>If the e-mail address used in your commits does not match the one in your github account, you will need to add and verify it in <a href="https://review.haiku-os.org/settings/#EmailAddresses">Gerrit E-Mail address settings</a>.</p>
+<a name="dev_access"></a>
+<h3>Git Access - Contributors and patch submitters</h3>
 
-<h4>Build Tools:</h4>
+So, you want to contribute to Haiku. That's awesome, thank you for your
+interest! This section will include instructions for setting up your local Git
+environment, an account on our Gerrit instance, as well as some guidelines on
+how to work with other contributors to get your changes included. This guide
+assumes that you are new to contributing to open-source projects. If that
+is not the case, you can skip some of the sections.
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
+<h4>A visual overview of our Git workflow</h4>
+<img src='/files/gitProcess_0.png' alt='A diagram describing the way our Git workflow works. This picture is there for aesthetic purposes and can be skipped. After a repository is cloned locally, if you have an idea, you can convert that idea into a commit using food (a pizza is depicted in the diagram). That commit will receive some change proposals, which will result in some further changes. The circle continues until the change is ready to be included in Haiku itself.'>
 
-```sh
-git clone "ssh://<login>@git.haiku-os.org/buildtools" && scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
-```
+<h4>Configuring Git locally<a name="configure_env"></a></h4>
 
-<h4>Haiku:</h4>
+<div class="alert alert-danger">
+<strong>This section is important!</strong> Failing to configure Git properly before a commit will result in an incorrect name being used in your commit, making it impossible to give you well-deserved credit for your work.</div>
```

Comment:
all the sections are... I think having these warnings everywhere just makes the information less concentrated and harder to find.